<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Pw">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>-90</x>
     <y>-200</y>
     <width>631</width>
     <height>861</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>631</width>
     <height>861</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>631</width>
     <height>861</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/cubee.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_Id">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;账 号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Icon_label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-120</y>
     <width>431</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/KSSA_Login_Title.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_Pw">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;密 码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_SignIn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
font: 11pt &quot;黑体&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Register">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>580</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;黑体&quot;;
border:none;</string>
   </property>
   <property name="text">
    <string>没有账号？点击注册</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Id">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_ip">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>630</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <zorder>Background</zorder>
  <zorder>lineEdit_Pw</zorder>
  <zorder>label_text_Id</zorder>
  <zorder>Icon_label</zorder>
  <zorder>label_text_Pw</zorder>
  <zorder>Button_SignIn</zorder>
  <zorder>Button_Register</zorder>
  <zorder>lineEdit_Id</zorder>
  <zorder>line_ip</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Button_Register</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>on_clicked_register()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>599</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_SignIn</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>on_clicked_signin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Pw</sender>
   <signal>returnPressed()</signal>
   <receiver>Login</receiver>
   <slot>on_clicked_signin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_clicked_register()</slot>
  <slot>on_clicked_signin()</slot>
 </slots>
</ui>
